test/storage.test.js

```diff
@@ -68,6 +69,15 @@ describe('Storage tests for feeds', () => {
     expect(feeds[2].link).toBe('');
     expect(feeds[3].link).toBe('http://seneca.co/jsmith');
   });
+
+  it('feed4 should not exist after being removed', async () => {
+    const feed = await getFeed(feed4.id);
+    await removeFeed(feed.id);
+    const removedFeed = await getFeed(feed.id);
+    const feeds = await getFeeds();
+    expect(removedFeed.id).toBe(undefined);
```

Comment:
This seems odd to me, that `removedFeed` would be an Object vs. `null`.  Can you confirm that this is actually returning something?  If it is, why is it?

Reply:
`console.log(removedFeed, typeof removedFeed, ...)`

Reply:
Confirmed on slack with the following image of `console.log`

![image](https://user-images.githubusercontent.com/18711727/78828964-896dc200-79b3-11ea-9bcb-4066e698cbd9.png)